pkg/sfu/buffer/buffer.go

```diff
@@ -431,7 +439,7 @@ func (b *Buffer) patchExtPacket(ep *ExtPacket, buf []byte) *ExtPacket {
 }
 
 func (b *Buffer) doFpsCalc(ep *ExtPacket) {
-	if b.frameRateCalculated || len(ep.Packet.Payload) == 0 {
+	if b.paused || b.frameRateCalculated || len(ep.Packet.Payload) == 0 {
```

Comment:
Passing along the `paused` state to buffer. Main reason being, when camera is disabled on Firefox, looks like we cannot fully disable it. It sends 1 fps at layer 2 when disabled. That is causing the FPS calculator to calculate wrong frame rate for layer 2.